### PR TITLE
docs - remove remaining refs to mapreduce

### PR DESCRIPTION
**Pull request by lisakowen**
a few more mapreduce refs to remove.

### CODE REVIEW

File: gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md [hunk 1]
```diff
@@ -895,7 +895,7 @@ Enable `LIMIT` operation to be performed while sorting. Sorts more efficiently w
 
 ## <a id="gp_external_enable_exec"></a>gp\_external\_enable\_exec 
 
- Activates or deactivates  the use of external tables that run OS commands or scripts on the segment hosts \(`CREATE EXTERNAL TABLE EXECUTE` syntax\). Must be enabled if using the Command Center or MapReduce features.
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md:898]: Is this not still required for GPCC?

**Review by dyozie — CHANGES_REQUESTED** [review]

File: gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md (hunk 1 above)
Comment (dyozie) [reply gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md:898]: Checked the gpcc docs.  Looks like it is required.

Comment (lisakowen) [reply gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md:898]: thanks for catching this.  i added that back in.

Comment (dyozie) [reply gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md:898]: Thanks!

**Review by dyozie — APPROVED** [review]

**Pull request merged**